example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -149,6 +149,13 @@ MM_CollectorLanguageInterfaceImpl::markingScheme_scanRoots(MM_EnvironmentBase *e
 		_markingScheme->markObject(env, rEntry->rootPtr);
 		rEntry = (RootEntry *)hashTableNextDo(&state);
 	}
+	OMR_VMThread *walkThread;
```

Comment:
Since the savedObject slot is now common shouldn't this code be common so that every language does not have to implement the same code?